docs/addons/addon-types.md

```diff
@@ -88,6 +88,10 @@ Use this boilerplate code while writing your own preset addon.
 
 <!-- prettier-ignore-end -->
 
-<div class="aside">
-Learn more about writing your own preset addon <a href="./writing-presets">here</a>.
-</div>
+## Learn more about the Storybook addon ecosystem
+
+- Addon types for other types of addons
```

Comment:
Could we simplify this to "Types of addons"?

Reply:
Sure we can and already adjusted  with the latest commit 🙏